tratto/src/main/java/star/tratto/identifiers/path/Path.java

```diff
@@ -0,0 +1,23 @@
+package star.tratto.identifiers.path;
+
+import java.nio.file.Paths;
+
+public enum Path {
```

Comment:
Path is a bad name for this class because Java has a class named Path.  How about OutputPath or the like?